mtgjson5/providers/tcgplayer.py

```diff
@@ -276,11 +478,48 @@ def generate_mtgjson_sealed_product_objects(
 
         mtgjson_sealed_products = []
 
+        with constants.RESOURCE_PATH.joinpath("sealed_name_fixes.json").open(
+            encoding="utf-8"
+        ) as f:
+            sealed_name_fixes = json.load(f)
+
+        with constants.RESOURCE_PATH.joinpath("sealed_name_fixes.json").open(
```

Comment:
This is loading the wrong file
```suggestion
        with constants.RESOURCE_PATH.joinpath("booser_box_size_overrides.json").open(
```